src/dev/create-extension.ts

```diff
@@ -0,0 +1,141 @@
+import arg from "arg";
+import path from "path";
+import fs from "fs";
+import { promisify } from "util";
+import { execa } from "execa";
+import ncp from "ncp";
+import { fileURLToPath } from "url";
+import { SOLIDITY_FRAMEWORKS } from "../utils/consts";
+import chalk from "chalk";
+
+const EXTERNAL_EXTENSIONS_DIR = "externalExtensions";
+const TARGET_EXTENSION_DIR = "extension";
+const TEMPLATE_FILE_SUFFIX = ".template.mjs";
+const BASE_PATH = "base";
+const SOLIDITY_FRAMEWORKS_PATH = "solidity-frameworks";
```

Comment:
nit: last two could be moved to consts since they used in `copy-template-files.ts` too

Reply:
Yup done https://github.com/scaffold-eth/create-eth/pull/73/commits/961251c7167bf3054aecad4981f72170046d3e34